stock_landed_costs_delivery/models/purchase_order.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def _prepare_slc_line_values(self):
```

Comment:
Use meaningful names:

```suggestion
    def _prepare_landed_cost_line_delivery_values(self):
```